src/listeners.c

```diff
@@ -112,6 +112,9 @@ void start_ex_listener(state_t *st) {
     case KEY_BACKSPACE:
       handle_backspace(st);
       break;
+    case KEY_ESC:
+      st->mode = NORMAL;
+      break;
```

Comment:
nice! but does it clear the status line when `ESC` is hit? I think in vim it does